sfge/src/test/java/com/salesforce/graph/build/MethodPathBuilderTest.java

```diff
@@ -154,7 +154,7 @@ public void testMethodWithSingleExpression() {
         MatcherAssert.assertThat(
                 path.verticesInCurrentMethod().get(1), instanceOf(ExpressionStatementVertex.class));
 
-        MatcherAssert.assertThat(getVerticesWithEndScope(), hasSize(equalTo(1)));
+        MatcherAssert.assertThat(getVerticesWithEndScope(), hasSize(equalTo(2)));
```

Comment:
The newly added synthetic vertices have an end scope, so all of the assertions in this class had to have their number incremented by 1.